specifyweb/frontend/js_src/lib/components/WbPlanView/CustomSelectElement.tsx

```diff
@@ -848,18 +848,18 @@ export function CustomSelectElement({
             className="sr-only bottom-0 top-[unset] flex w-full justify-center"
           >
             <input
+              id={id('validation')}
+              // Act as an error message, not an input
               defaultValue={validation}
+              // Announce validation message to screen readers
+              aria-live="polite"
               /*
                * Set a validation message for input (using useValidation).
                * It will be displayed by browsers on form submission
                */
-              ref={validationRef}
-              // Announce validation message to screen readers
-              aria-live="polite"
-              // Act as an error message, not an input
               role="alert"
               // Associate validation message with the listbox
-              id={id('validation')}
+              ref={validationRef}
```

Comment:
looks like comments are now out of order after eslint rule that sorts attributes - could you please reorder the comments so that they are attached to the attributes they were attached to before?